config/settings.yml

```diff
@@ -38,6 +38,15 @@ user:
     - sulair:proxy-access
     - lane:proxy-access
   locations:
+    # Codes from https://github.com/sul-dlss/cocina-models/blob/8fc7b5b9b0e3592a5c81f4c0e4ebff5c926669c6/openapi.yml#L1330-L1339
+    # labels from https://consul.stanford.edu/display/chimera/Rights+Metadata+Locations
+    labels:
+      spec: 'Special Collections reading room'
+      music: 'Music Library - main area'
+      ars: 'Archive of Recorded Sound listening room'
+      art: 'Art Library'
+      hoover: 'Hoover Library'
+      m&m: 'Media & Microtext'
```

Comment:
YAML question: does this hash key come through OK? Might be worth tweaking a test to make sure, or just running a manual test?

Reply:
I'll merge this and run a test myself, nevermind!